index.html

```diff
@@ -1,14 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Hello Microverse</title>
+    <title>Temesgen Portfolio</title>
     <link rel="stylesheet" href="style.css">
 
 </head>
+
 <body>
-    <h1 class="main-header">Hello Microverse!</h1>
+    <header>
+        <h4>My Logo</h4>
+        <nav>
+            <a href="">
+            </a>
+            <img src="./img/menu.png" alt="web menu graphics">
+        </nav>
+    </header>
+
+    <main>
+        <section class="Profile">
+                <h1>I’m Temesgen <br> Glad to see you!</h1>
+                <p>
+                    I’m a software developer! I can help you
+                    build a product, feature or website. Look
+                    through some of my work and experirience!
+                    If you like what you see and have a project
+                    you need coded, don't hestitate to contact me.
+                </p>
+            <h4>Lets connect</h4>
+            <ul class="social-handle">
+                <li><a href="#"><img src="./img/twitter.png" alt="twitter icon"></a></li>
+                <li><a href="#"><img src="./img/Linkedin.png" alt="linkedin icon"></a></li>
+                <li><a href="#"><img src="./img/medium.png" alt="medium icon"></a></li>
+                <li><a href="#"><img src="./img/github.png" alt="github icon"></a></li>
+                <li><a href="#"><img src="./img/angellist.png" alt="angellist icon"></a></li>
+            </ul>
```

Comment:
As required by the previous reviewer, please update the design to stick to Figma.

Please remove the dot before each image.
 
You can do this to correct it :
 
`ul{list-style: none;}
`
Please add space between each image and change their size

Your design:

<img width="217" alt="image" src="https://user-images.githubusercontent.com/7243132/170825644-def05599-7b22-44f4-b534-163ceaf5ad92.png">

FIgma
<img width="302" alt="image" src="https://user-images.githubusercontent.com/7243132/170825745-6bab808e-6af5-4676-80e6-19274f204bfc.png">